### DISCUSSION TITLE
Keep restarting mariadb docker container

### DISCUSSION THREAD

**Discussion by wpdev00018** [EasyEngine/easyengine#1848] [General]
Is there any solution to fix this issue on the exiting site?

It's similar to this type issue : https://community.easyengine.io/t/easyengine-mariadb-v4-0-0-keeps-restarting/13243

Thanks

**Comment by marcoandrei** [comment]
@wpdev00018,

Have you tried to recreate the site with `local-db` option (so EE4 creates a separate db container)? (I'm not sure if it is an option.)

**Comment by marcoandrei** [comment]
@wpdev00018 

I found this conversation that maybe can help you: https://github.com/EasyEngine/easyengine/discussions/1759#discussioncomment-4505933
